service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/SendAbsentAttachmentTaskDefinition.java

```diff
@@ -0,0 +1,32 @@
+package uk.nhs.adaptors.gp2gp.ehr;
+
+import static uk.nhs.adaptors.gp2gp.common.task.TaskType.SEND_ABSENT_ATTACHMENT;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.experimental.SuperBuilder;
+import lombok.extern.jackson.Jacksonized;
+import uk.nhs.adaptors.gp2gp.common.task.TaskType;
+
+@Jacksonized
+@Getter
+@SuperBuilder
+@EqualsAndHashCode(callSuper = true)
+public class SendAbsentAttachmentTaskDefinition extends DocumentTaskDefinition {
+
+    public SendAbsentAttachmentTaskDefinition(DocumentTaskDefinitionBuilder<?, ?> b, String title) {
```

Comment:
Pattern is generally to use builders on constructors, is this constructor neccessary?

Reply:
Found that weird too, but it was a resolution to InvalidDefinitionException being thrown